Examples/SimFit/SimFit.cpp

```diff
@@ -224,7 +217,8 @@ energyPar createEnergyPar(size_t NEvents, double SqrtS, int seed,
 
   auto data =
       ComPWA::Data::generate(NEvents, kin, RandomGenerator, amp, mcSample);
-  return energyPar(NEvents, std::move(kin), amp, data, mcSample);
+//  return energyPar(NEvents, std::move(kin), amp, data, mcSample);
```

Comment:
Could you remove the comment. I prefer to just remove code instead of commenting